package-builders/Dockerfile.centos8

```diff
@@ -0,0 +1,58 @@
+FROM centos:8
+
+ARG VERSION=0.1
+
+ENV VERSION=$VERSION
+
+ADD package-builders/fedora-build.sh /build.sh
+
+RUN yum install -y 'dnf-command(config-manager)'
+RUN yum config-manager --set-enabled PowerTools
+RUN yum update -y
+RUN yum install -y epel-release
+RUN yum install -y http://mirror.centos.org/centos/8/PowerTools/x86_64/os/Packages/Judy-devel-1.0.5-18.module_el8.1.0+217+4d875839.x86_64.rpm
```

Comment:
Is there some reason we're installing this by URL even after enabling the PowerTools repo?

Reply:
Yes because their repo is _somewhat_ broken for _some_ packages. I know its all very strange; I get the impression they kind of "rushed" this. I should probably file a bug upstream for this or find one/etc.

Reply:
Added a comment in the `Dockerfile`